shared/src/main/scala/amf/core/internal/plugins/document/graph/parser/GraphParserHelpers.scala

```diff
@@ -69,16 +69,9 @@ trait GraphParserHelpers extends GraphContextHelper {
       case _ => node.as[YScalar].text
     }
 
-  val fieldsWithId: Set[Field] = Set(
```

Comment:
This is shady.

Reply:
Yes, it is. Using a fixed list is shady. We could not find why is this list present, we suppose it was to limit the original change for fear of breaking something. There is no track in the issue or in the commit message of why this list exists.